### PR TITLE
Create metro.json

### PR DESCRIPTION
**Pull request by Haroenv**
so the site isn't live yet, but this should be the config since the site is the same as Jest

site: http://facebook.github.io/metro or http://facebook.github.io/metro-bundler

cc @cpojer

### CODE REVIEW

**Comment by cpojer** [comment]
That's awesome! Can I send a PR for this repo when we rename the website from metro-bundler to metro? :)

**Comment by Haroenv** [comment]
Yep, you can send a PR to this repo with any changes in the configuration. I put metro for now, but if metro-bundler will launch earlier, that can be changed

**Comment by cpojer** [comment]
Oh, there will only be one. It'll start as metro-bundler and then we'll rename that to metro. Unfortunately the renaming takes a while internally.. :(

File: configs/metro.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "index_name": "jest",
```
Comment (s-pace) [inline configs/metro.json]: Typo

File: configs/metro.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "start_urls": [
+    {
+      "url": "http://facebook.github.io/metro-bundler/docs/",
```
Comment (s-pace) [inline configs/metro.json]: They plan to change this

**Comment by s-pace** [comment]
Hi again @cpojer,
@Haroenv has shown you the right process. Please do so as soon as you are ready from your site. I will double check it/ tailored/ deployed/provide you the snippet as soon as you are live.
Thank you guys

File: configs/metro.json (hunk 2 above)
Comment (Haroenv) [reply configs/metro.json]: good call, let's use the one that's online first

**Comment by cpojer** [comment]
Site is live here: https://facebook.github.io/metro-bundler/ it only contains the scaffolding for the docs and we'll fill it out later.

**Comment by s-pace** [comment]
@cpojer 

Congratulations, your search is now ready!
I've successfully configured the underlying crawler and it will now run every 24h.

You're now a few steps away from having it working on your website:
- Copy the following CSS/JS snippets and add them to your page


<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />

<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
<script type="text/javascript"> docsearch({
  apiKey: '0cf53e80b4f3b371219cdcaa6b62e08b',
  indexName: 'metro',
  inputSelector: '### REPLACE ME ####',
  debug: false // Set debug to true if you want to inspect the dropdown
});
</script>

- Add a search input if you don't have any yet, and update the inputSelector value in the code snippet to a CSS selector that targets your input field.
- Optionally customize the look and feel by following the DocSearch documentation (https://community.algolia.com/docsearch/documentation/)
- You can also check your configuration in the github repo (https://github.com/algolia/docsearch-configs/blob/master/configs/metro.json). Feel free to open pull requests!

Feel free to get back to us if you have any issues or questions regarding the integration.
We'd also be happy to get your feedback and thoughts about DocSearch - so we can continue to improve it.

Have a nice day :)

**Comment by cpojer** [comment]
Thank you very much.

**Comment by s-pace** [comment]
You are welcome, If you need to add an new language, you will only need to add it into the lang array.
These "language" variables will also behave as arguments for faceting. Thus you can filter which kind of results you want to request. This will be made from youir snippet by adding the following input: 
`algoliaOptions: { facetFilters: ["lang:en"] } `

**Comment by Haroenv** [comment]
you can also do that on jest by the way, which already has different languages, but the search isn't enabled for the others yet @cpojer

**Pull request merged**